.github/CODEOWNERS

```diff
@@ -353,6 +353,8 @@
 
 # AzureSdkOwners: @kashifkhan
 # ServiceLabel: %Service Bus
+# ServiceOwners: @kashifkhan
```

Comment:
do we need `ServiceOwner`? Im definitely not on the service side :)

Reply:
Do we need "service attention" label?

Reply:
nope we dont.